packages/recipe/src/components/EzTextStyle/Documentation/Stories/Default.stories.tsx

```diff
@@ -11,5 +34,13 @@ export default meta;
 type Story = StoryObj<typeof EzTextStyle>;
 
 export const Default: Story = {
-  render: () => <div>Coming soon...</div>,
+  args: {},
+  parameters: {
+    playroom: {
+      code: dedent`
+        <EzTextStyle>This is some example text.</EzTextStyle>
+      `,
+    },
+  },
```

Comment:
```suggestion
    playroom: {code: <EzTextStyle>This is some example text.</EzTextStyle>},
```
No `dedent` needed here. It's only needed if there is more than one line.